app/api/email_utils.py

```diff
@@ -149,3 +149,5 @@ def send_email_new_request(user_sender, user_recipient, notes, sender_role):
     )
     subject = "Mentorship System - You have got new relation request"
     send_email(user_recipient.email, subject, html)
+
```

Comment:
can you undo these changes? just to keep the changes to test module?

Reply:
can you remove these changes? I can merge them but if you could take these off as well. This seems like style format changes, that can be done in another PR